components/lib/multiselect/MultiSelect.js

```diff
@@ -873,7 +890,10 @@ export const MultiSelect = React.memo(
                     const iconProps = mergeProps(
                         {
                             className: cx('removeTokenIcon'),
-                            onClick: (e) => removeChip(e, val)
+                            onClick: (e) => removeChip(e, val),
+                            tabIndex: props.tabIndex || '0',
+                            'aria-label': localeOption('removeTokenIcon'),
+                            onKeyDown: (e) => onremoveTokenIconKeyDown(e, val)
```

Comment:
Can you move the onKeyDown up so its next to OnClick?  I like all the `on`'s grouped together

Reply:
Done @melloware